### PR TITLE
Federation amalgamation

### PR DESCRIPTION
**Pull request by JaswiniP**
Added ids related to amalgamation and federation group

### CODE REVIEW

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml [hunk 1]
```diff
@@ -32,8 +32,8 @@
             @foreach (var l in Model.LinkedEstablishments)
             {
                 <tr class="govuk-table__row">
-                    <td data-label="URN" class="govuk-table__cell">@Html.ActionLink(l.Urn.ToString(), "Details", new {id = l.Urn})</td>
-                    <td data-label="Establishment name" class="govuk-table__cell">@Html.ActionLink((l.EstablishmentName ?? "-"), "Details", new {id = l.Urn})</td>
+                    <td data-label="URN" id=@l.Urn  class="govuk-table__cell">@Html.ActionLink(l.Urn.ToString(), "Details", new {id = l.Urn})</td>
```
Comment (RogerHowellDfE) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml:35]: Does this `id` value need quotes around the value?

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Create.cshtml [hunk 2]
```diff
@@ -327,8 +327,8 @@
                         <p tabindex="0" id="model-desc">The record you are creating for <strong>@(warning.MessageParameters[0])</strong> in <strong>@(warning.MessageParameters[1])</strong> may duplicate existing record(s): <strong>@(warning.MessageParameters[2])</strong>
                     </div>
                     <div class="button-row">
-                        <button type="submit" name="@nameof(Model.ActionStep)" value="@Model.ActionStep" class="govuk-button">Proceed</button>
-                        <button type="submit" name="@nameof(Model.ActionStep)" value="@Model.PreviousStep" class="govuk-button govuk-button--secondary" >Cancel</button>
+                        <button type="submit" id="proceed-button" name="@nameof(Model.ActionStep)" value="@Model.ActionStep" class="govuk-button">Proceed</button>
+                        <button type="submit" id="cancel-button" name="@nameof(Model.ActionStep)" value="@Model.PreviousStep" class="govuk-button govuk-button--secondary" >Cancel</button>
```
Comment (RogerHowellDfE) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Create.cshtml:331]: I note that there are multiple elements with the same `id` -- if you're happy that they'll never be on the same page together, I'm happy :)

File: Web/Edubase.Web.UI/package-lock.json [hunk 3]
```diff
@@ -6291,1037 +7304,1191 @@
...
+      "version": "20.2.9",
+      "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-20.2.9.tgz",
+      "integrity": "sha512-y11nGElTIV+CT3Zv9t7VKl+Q3hTQoT9a1Qzezhhl6Rp21gJ/IVTW7Z3y9EWXhuUBC2Shnf+DX0antecpAwSP8w==",
+      "dev": true,
+      "engines": {
+        "node": ">=10"
+      }
     },
-    "zwitch": {
+    "node_modules/zwitch": {
```
Comment (RogerHowellDfE) [inline Web/Edubase.Web.UI/package-lock.json:8483]: It's probably fine, but I don't know enough to know the broader impact of editing the lock file like this.

If it's accidental, maybe the changes could be removed from this PR? Or perhaps another dev has a view on updating lock files?

File: Web/Edubase.Web.UI/package-lock.json [hunk 4]
```diff
@@ -6291,1037 +7304,1191 @@
...
         "signal-exit": "^3.0.2",
         "typedarray-to-buffer": "^3.1.5"
       }
     },
-    "yallist": {
-      "version": "2.1.2",
-      "resolved": "https://registry.npmjs.org/yallist/-/yallist-2.1.2.tgz",
-      "integrity": "sha1-HBH5IY8HYImkfdUS+TxmmaaoHVI=",
+    "node_modules/yallist": {
+      "version": "3.1.1",
```
Comment (RogerHowellDfE) [inline Web/Edubase.Web.UI/package-lock.json:8460]: Moving to more recent versions is a good thing, but it's something to double check if these are accidental /  deliberate changes?

_(changes to the lock file might have happened from using a more recent version of node?)_

File: Web/Edubase.Web.UI/package-lock.json (hunk 3 above)
Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/package-lock.json:8483]: Changes to the package lock are normally part of an automated process, when working with NPM

File: Web/Edubase.Web.UI/package-lock.json (hunk 4 above)
Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/package-lock.json:8460]: Changes to the package lock are normally part of an automated process, when working with NPM

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml (hunk 1 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml:35]: when I didn't use the quotes it was working as expected. I tried using quotes the results are same. Happy to change it either ways

Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml:35]: The @ symbol means we are going into C#/Razor code at this point , the id will be set to the Urn of l

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Create.cshtml (hunk 2 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Create.cshtml:331]: Yes. proceed, cancel, submit these buttons are seen in every page but not seen on the same page together

**Review by ScottDawsonDfE — APPROVED** [review]

File: Web/Edubase.Web.UI/package-lock.json (hunk 4 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/package-lock.json:8460]: I'm not aware of this. Will make sure about this in future

**Pull request merged**